R/health-plots.R

```diff
@@ -169,33 +203,52 @@ inc_cum <- function(inc, data_mean_weighted_inc_wide) {
                     stroke = "Stroke incidence")
 
   y_value <- switch(inc,
-                    asthma = "cumdiff_asthma",
-                    ckd = "cumdiff_ckd",
-                    diabetes = "cumdiff_diabetes",
-                    ischemia = "cumdiff_ihd",
-                    stroke = "cumdiff_stroke")
+                    asthma = "diff_inc_asthma_mean",
+                    ckd = "diff_inc_ckd_mean",
+                    diabetes = "diff_inc_db_mean",
+                    ischemia = "diff_inc_ihd_mean",
+                    stroke = "diff_inc_stroke_mean")
+
```

Comment:
Sorry, I should have named the variables in as cumdiff, otherwise it's confusing.
I'll put suggestions to rename all the variables in this script and data-process.R.